macros/contexts/contextBoolean.pl

```diff
@@ -22,22 +26,22 @@ =head2 CONSTANTS
     $T = Context()->T;
     $T = context::Boolean->T;
 
-=head2 VARIABLES
+=head1 VARIABLES
 
 By default, this context has two variables, C<p> and C<q>. More variables can be added through the usual
 means of modifying context:
 
     Context->variables->add( r => 'Boolean' );
 
-=head2 OPERATORS
+=head1 OPERATORS
 
 Changing the LaTeX representations of the boolean operators is handled through the operators C<or>, C<and>,
 C<xor>, and C<not>. Note the extra space following the LaTeX command.
 
     Context->operators->set( not => { TeX => '\neg ' } );
 
 
-=head3 Aliases and Alternatives
+=head1 Aliases and Alternatives
```

Comment:
This should be `head2`.  This is supposed to be a sub header under the `head1 OPERATORS` above.

In general, it may be best if you just revert about all of the changes to this file (maybe other than the addition of a `SYNOPSIS`.  It had a decent POD structure to begin with.